tests/integration/collective/examples/test_collective.py

```diff
@@ -0,0 +1,127 @@
+# Copyright 2023 NVIDIA Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import pytest
+from collective import (
+    collective_test,
+    collective_test_matvec,
+    create_int64_store,
+)
+
+
+def test_no_collective() -> None:
```

Comment:
formatting of this file is awful. I think we should go check the formatter configuration.